src/components/Form/MucCounter.vue

```diff
@@ -0,0 +1,128 @@
+<template>
+  <div class="wrapper">
+    <MucButton
+      v-on:click="clickedMinus"
+      variant="secondary"
+      :disabled="disableMinus"
+    >
+      <template #default><muc-icon icon="minus" /></template>
+    </MucButton>
+    <p>
+      <strong
+        class="centered-text"
+        style="color: #005a9f"
```

Comment:
You can use this variable instead: https://github.com/it-at-m/muc-patternlab-vue/blob/beta/public/assets/temporary/custom-style.css#L6

Reply:
Done.